electron/golem_handler.js

```diff
@@ -43,21 +44,21 @@ class GolemProcess {
 
         /* Handle process events */
         this.process.on('error', data => {
-            console.error('💻 Cannot start Golem:', data.toString())
+            log.error('MAIN_PROCESS > GOLEM_HANDLER', 'Cannot start Golem:', data.toString())
         });
         this.process.on('exit', code => {
-            console.log('💻 Golem exited with code', code);
+            log.info('MAIN_PROCESS > GOLEM_HANDLER', 'Golem exited with code', code)
         });
-        /* FIXME: we shouldn't be catching stdout here.
-        this.process.stderr.on('data', data => {
-            console.error('💻 Golem error:', data.toString());
-        });*/
+    /* FIXME: we shouldn't be catching stdout here.
+    this.process.stderr.on('data', data => {
```

Comment:
We need to log Golem's output if it fails to start. At this point, Golem might not even have a chance to create it's own log files.